aries-site/src/pages/components/layer/index.mdx

```diff
@@ -0,0 +1,247 @@
+
+### When to use double confirmation
+
+When a user attempts to close out of a layer, the layer closing behavior should always avoid losing user's work.
+
+When data has been edited or changed and the user tries to close the layer, a double confirmation should be shown.
+
+| Layer state                             | Confirmation type   | Notes                                                                                                                                                                                                                      |
+| --------------------------------------- | ------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| User has not input or changed any data. | No confirmation     | The user will not lose anything if the layer closes immediately.                                                                                                                                                           |
+| User has input or changed data.         | Double confirmation | The double confirmation should show when the user clicks the "X" or Cancel button, clicks outside of the layer, or presses the Escape key. This short confirmation ensures that user's work won't unintentionally be lost. |
+
+### Double confirmation anatomy
+
+A double confirmation appears as a center layer. This placement puts the confirmation at the center of the user's attention, reinforcing the importance of the dialog's prompt and minimizing distractions from other page content.
+
+<Example plain>
+  <DoubleConfirmationAnatomy />
+</Example>
+
+| Label  | Region       | Purpose                                                              | Required? | Notes                                                                                                      |
+| ------ | ------------ | -------------------------------------------------------------------- | --------- | ---------------------------------------------------------------------------------------------------------- |
+| **1a** | **Title**    | Explains that data will be discarded when the user closes the layer. | required  | --                                                                                                         |
+| **1b** | **Subtitle** | To clarify that changes will not be saved.                           | required  | --                                                                                                         |
+| **2**  | **Footer **  | Contains two actions: "Cancel" and "Discard"                         | required  | The actions should be right-aligned with primary action on the right, to align with center layer guidance. |
+
+### Double confirmation example
```

Comment:
Disregard if this example is still in a WIP

The "add application" layer here doesn't seem to behave correctly for me. I can't scroll within the layer but I can still scroll through the content behind it. Also looks like the filters button doesn't have content.

Reply:
Yeah that's a WIP still.

Reply:
For now, just having the layer open with respect to the whole window. There seems to be a bug with the layer scroll when the layer is restricted to the example container. Planning to debug that separately from this PR.